### ISSUE TITLE
Case option not applied to type names

### ISSUE THREAD

**Issue by miqueljuan** [moh-hassan/odata2poco#28]
Is there a possibility to apply -c option  to poco class names, enum type names? For properties works as expected.

Thanks for your work!!

**Comment by moh-hassan** [comment]
Welcome to Odata2Poco project.
Case option for type names will be available in the next release (like properties).

**Comment by moh-hassan** [comment]
A new package v3.3.0 is released that fix your issue.

The [new features](https://github.com/moh-hassan/odata2poco#what-is-new-in-v330):

You can download:
- Commandline console: [o2pgen](https://www.nuget.org/packages/OData2Poco.CommandLine/3.3.0)
- Dotnet core [tool](https://www.nuget.org/packages/OData2Poco.dotnet.o2pgen/3.3.0)
- class [library](https://www.nuget.org/packages/OData2Poco/3.3.0):

Try the filter option using --include
```bash
o2pgen -r http://services.odata.org/V4/TripPinServiceRW -v --entity-case camel
```
How to use: read [wiki](https://github.com/moh-hassan/odata2poco/wiki/CommandLine-Reference)

Let me know if that version fixes your issue.

**Comment by miqueljuan** [comment]
Hi @moh-hassan ,

I have tried the new option --entity-case pas and works as a charm for entity names, but for navigation properties that references this entities the name of the type is not pascal-case causing compiling errors.

For example:

```csharp
public class Address{
       public string Name {get;set;}
}
public class Contact{
      public address Address {get;set;}
}
```
Thanks!!

**Comment by moh-hassan** [comment]
Thanks @miqueljuan for feedback.
I'll start fixing the issue.

**Comment by moh-hassan** [comment]
The issue is fixed in [v3.3.1](https://www.nuget.org/packages/OData2Poco.CommandLine/3.3.1).
Both Navigation properties and BaseClass(Parent) are fixed.

**Comment by miqueljuan** [comment]
Thanks @moh-hassan , works perfectly!!
